Files/BaseLayout.cs

```diff
@@ -341,22 +341,21 @@ protected IEnumerable<ListedItem> GetAllItems()
 
         public virtual void ResetItemOpacity()
         {
-            IEnumerable items = GetAllItems();
+            IEnumerable<ListedItem> items = GetAllItems();
             if (items == null)
             {
                 return;
             }
 
-            foreach (ListedItem listedItem in items)
+            for (int i = 0; i < items.Count(); i++)
```

Comment:
This would result in count calculation for each loop iteration. You may want to call `ToList()` first.